app/services/select_preferable_service.rb

```diff
@@ -0,0 +1,13 @@
+class SelectPreferableService
+
+  def initialize(params)
+    @params = params
+  end
+
+  def select_preferable
+    if @params.values_at('preferable').join.to_i == 1
```

Comment:
are you sure that here is `1` value? Not `true` or `false` ?

Reply:
It's `0` or `1` due to the simple_form